go.mod

```diff
@@ -4,25 +4,95 @@ go 1.17
 
 require (
 	github.com/cloudevents/sdk-go/v2 v2.5.0
+	github.com/google/uuid v1.3.0
+	github.com/imroc/req v0.3.2
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/keptn/go-utils v0.12.0
+	github.com/keptn/go-utils v0.13.0
+	github.com/keptn/kubernetes-utils v0.13.0
+	github.com/mitchellh/mapstructure v1.1.2
+	github.com/prometheus/common v0.10.0
```

Comment:
not sure why it is using prometheus/common as a direct dependency here...

Btw did you upgrade to Go 1.17 locally?

Reply:
It seems like I used the wrong `log` package for a function. The dependency should be removed now. 

Also, I'm currently using Go 1.18 on my local machine.